vertx-web/src/main/java/io/vertx/ext/web/handler/impl/ErrorHandlerImpl.java

```diff
@@ -95,9 +92,11 @@ private boolean answerWithKnownFailure(RoutingContext context) {
     
     if(failure instanceof HeaderTooLongException){
       // Math.random is random enough for this purpose. Will wait between 1 ms and 11ms
-      answerDelayTimer.schedule(
-          new DelayedAnswer(context, 400, "A header was too long to process"), 1 + (long)(Math.random() * 10)
+      
```

Comment:
I think that we should have some deterministic code and remove the timer. If one needs to protect from a DDoS then it will probably be done before a request even reaches this code...